doc/rbd/rbd-mirroring.rst

```diff
@@ -238,19 +287,73 @@ For example::
 .. tip:: You can enable journaling on all new images by default by adding
    ``rbd default features = 125`` to your Ceph configuration file.
 
-Enable Image Mirroring
-----------------------
+Create Image Mirror-Snapshots
+-----------------------------
 
-If the mirroring is configured in ``image`` mode for the image's pool, then it
-is necessary to explicitly enable mirroring for each image within the pool.
-To enable mirroring for a specific image with ``rbd``, specify the
-``mirror image enable`` command along with the pool and image name::
+When using snapshot-based mirroring, mirror-snapshots will need to be created
+whenever it is desired to mirror the changed contents of the RBD image. To
+create a mirror-snapshot manually with ``rbd``, specify the
+``mirror image snapshot`` command along with the pool and image name::
+
+        rbd mirror image snapshot {pool-name}/{image-name}
+
+For example::
+
+        $ rbd --cluster site-a mirror image snapshot image-pool/image-1
+
+By default only ``3`` mirror-snapshots will be created per-image. The most
+recent mirror-snapshot is automatically pruned if the limit is reached prior
+to creating a new mirror-snapshot. The limit can be overridden via the
+``rbd_mirroring_max_mirroring_snapshots`` configuration option if required.
+Additionally, mirror-snapshots are automatically deleted when the image is
+removed or when mirroring is disabled.
+
+Mirror-snapshots can also be automatically created on a periodic basis if
+mirror-snapshot schedules are defined. The mirror-snapshot can be scheduled
+globally, per-pool, or per-image levels. Multiple mirror-snapshot schedules can
```

Comment:
Did you intentionally skipp the namespace level?

Reply:
Yeah, I hesitated about adding it since we didn't talk about namespaces anywhere else and I didn't want to open a can-of-worms.